diff --git a/nuxt.config.ts b/nuxt.config.ts
index aa95faa5..69d6ccda 100644
--- a/nuxt.config.ts
+++ b/nuxt.config.ts
@@ -75,7 +75,8 @@ export default defineNuxtConfig({
       // PASTE_API defaults to https://paste.aosc.io. To use the proxy,
       // please set PASTE_API=http://localhost:2334 in your environmental
       // variables or the `.env` file.
-      pasteApi: process.env.PASTE_API || 'https://paste.aosc.io'
+      //pasteApi: process.env.PASTE_API || 'https://paste.aosc.io'
+      pasteApi: '/pasteApi'
     }
   },
 
@@ -90,6 +91,11 @@ export default defineNuxtConfig({
     },
     server: {
       proxy: {
+        '/pasteApi': {
+          target: 'https://paste.aosc.io',
+          changeOrigin: true,
+          rewrite:  (path) => path.replace(/^\/pasteApi/, '')
+        },
         '/galleryFile': {
           target: 'https://aosc.io',
           changeOrigin: true
